### ISSUE TITLE
LOCKSS permission statement is wrong

### ISSUE THREAD

**Issue by mpbraendle** [pkp/pkp-lib#4931]
templates/gateway/lockss.tpl, line 120:

<p>LOCKSS system has permission to ingest, preserve, and serve this Archival Unit.</p>

Correct should be, according to the LOCKSS Guidelines, https://lockss.github.io/web-harvest-guidelines

<p>LOCKSS system has permission to collect, preserve, and serve this Archival Unit.</p>

(Instead, the CLOCKKS statement was used for LOCKSS, which is wrong, see also https://clockss.org/wp-content/uploads/2018/08/Web_Harvest_Guidelines_-_CLOCKSS.pdf )

Effect of the wrong statement : LOCKSS can't harvest articles.

**What application are you using?**
OJS 3.1.2-1

**Comment by mpbraendle** [comment]
Indeed, after we have corrected the statement, LOCKSS clients do harvest now our content. Before, the following error was reported by LOCKSS: No permission statement on permission page.

**Comment by kaitlinnewson** [comment]
Resolved back in https://github.com/pkp/ojs/pull/2433 and confirmed that this is fixed in 3.3+.
